plutus-benchmark/README.md

```diff
@@ -0,0 +1,67 @@
+## Plutus Benchmarks
```

Comment:
:100:

Reply:
I had to write all that down because I can _never_ remember the correct stack/cabal incantations to run things.